evergreen.yml

```diff
@@ -154,6 +163,20 @@ functions:
             cargo set-version $RELEASE_VERSION
           fi
 
+  "sign windows":
+    - command: shell.exec
+      params:
+        working_dir: mongosql-odbc-driver/installer/msi
+        script: |
+          echo "${signing_token_odbc_driver}" > ./signing_auth_token
+          /usr/local/bin/notary-client.py \
+              --key-name "bi-connector" \
+              --auth-token-file ./signing_auth_token \
+              --comment "Evergreen Automatic Signing (Atlas SQL ODBC) - ${version_id} - ${build_variant}" \
+              --notary-url "${notary_client_url}" \
+              --skip-missing \
```

Comment:
It seems odd to skip missing files (from the python script : "Skip missing files, instead of exiting with an error"). If the msi file is not there, it seems like a good idea to fail the signing task, isn't it?

Reply:
I have no idea what skip-missing means, I have no suggestion that it means skip missing files.